docs/general/dql/selects.rst

```diff
@@ -685,9 +684,9 @@ This behaviour applies to:
     logic`_. For better performance, consider using the :ref:`ignore3vl
     <scalar-ignore3vl>` function.
 
-    Additionally, When using ``NOT`` with ``LIKE ANY`` or ``NOT LIKE ANY``, the
-    default maximum size of the array can be 8192. To use larger arrays, you
-    must configure the :ref:`indices.query.bool.max_clause_count
+    Additionally, When ``NOT`` with ``LIKE ANY`` or ``NOT LIKE ANY`` on
+    arrays causes ``TooManyClauses`` errors, you should consider increasing
```

Comment:
```suggestion
    arrays causes ``TooManyClauses`` errors, you could consider increasing
```
I think `could` is more accurate, as we should not encourage users to increase the limit, but rather consider rewriting their queries or even have a better schema where those nested `IN()` subqueries are not used.